src/compressor/zlib/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 # zlib
+set(isal_src_dir ${CMAKE_SOURCE_DIR}/src/isa-l)
+include_directories(${isal_src_dir}/include)
```

Comment:
please don't add include globally. add it for those targets which actually `#include` them.

Reply:
@tchaikov 
I'll follow your suggestion.
This commit follows the below usage in erasure-code:
https://github.com/ceph/ceph/blob/master/src/erasure-code/isa/CMakeLists.txt#L3